src/main/resources/data/extrastorage/recipes/raw_neural_processor.json

```diff
@@ -7,7 +7,7 @@
   ],
   "key": {
     "a": {
-      "item": "minecraft:crafting_table"
+      "tag": "forge:workbenches"
```

Comment:
This tag doesn't work

Reply:
Oh sorry about that, I remembered it as added by Forge, removed :)